deploy/auto-install.sh

```
@@ -190,19 +187,19 @@ upgrade_docker_openwisp() {
 
 	start_step "Configuring docker-openwisp..."
-	for config in $(grep '=' $ENV_BACKUP | cut -f1 -d'='); do
+	for config in $(grep '=' "$ENV_BACKUP" | cut -f1 -d'='); do
 		value=$(get_env "$config" "$ENV_BACKUP")
 		set_env "$config" "$value"
 	done
 	report_ok
 
 	start_step "Starting images docker-openwisp (this will take a while)..."
-	make start TAG=$(cat $INSTALL_PATH/VERSION) -C $INSTALL_PATH/ &>>$LOG_FILE
+	make start TAG=$(cat "$INSTALL_PATH/VERSION") -C "$INSTALL_PATH/" &>>"$LOG_FILE"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)